test/integration/aws/cognito-user-pool.test.js

```diff
@@ -64,6 +75,7 @@ describe('AWS - Cognito User Pool Integration Test', function () {
     return BbPromise.all([
       deleteUserPool(poolExistingSimpleSetup),
       deleteUserPool(poolExistingMultiSetup),
+      deleteUserPool(poolExistingCustomEmailSenderSetup),
```

Comment:
Did you test that these integration tests are now passing as well? Or how did you validate these changes?

Reply:
I did confirm these integration tests are passing using `AWS_ACCESS_KEY_ID=XXX AWS_SECRET_ACCESS_KEY=xxx npx mocha test/integration/aws/cognito-user-pool.test.js`.

Reply:
perfect 👍